class/hd-segwit-bech32-wallet.js

```diff
@@ -210,7 +215,8 @@ export class HDSegwitBech32Wallet extends AbstractHDWallet {
     const keypairs = {};
     const values = {};
 
-    inputs.forEach(input => {
+    for (let i = 0; i < inputs.length; i++) {
```

Comment:
why not to use map?

Reply:
because you need to wait for async function `_getPubkeyByAddress` few lines below

Reply:
Why? If map's callback is `async` it will wait for `await` to be resolved before checking the next iteration.

Reply:
not it won't, check this code and the result 
`const wait = i =>
  new Promise(resolve => {
    setTimeout(() => {
      resolve();
    }, 5000/i);
  });

const testArr = [1, 2, 3, 4, 5];

const test = () => {
  console.log('Start');
  testArr.map(async el => {
    await wait(el);
    console.log('el', el);
  });
  console.log('Stop');
};

test();`

<img width="582" alt="Zrzut ekranu 2020-06-29 o 14 36 20" src="https://user-images.githubusercontent.com/67273035/86006440-3f2a3900-ba16-11ea-95b2-c0e7e3175b44.png">